### PR TITLE
updated troubleshooting

### PR DESCRIPTION
**Pull request by Plamen5kov**

### CODE REVIEW

**Comment by Plamen5kov** [comment]
@ikoevska

File: troubleshooting.md [hunk 1]
```diff
@@ -16,6 +16,20 @@ NativeScript {{site.current-version}} is known to have the following issues and
 
 ### Problem
 
+When trying to run an android project with cli an error commes up:
```
Comment (ikoevska) [inline troubleshooting.md]: When you attempt to run any build-related operations for Android in the NativeScript CLI, the following message appears.

File: troubleshooting.md [hunk 2]
```diff
@@ -16,6 +16,20 @@ NativeScript {{site.current-version}} is known to have the following issues and
...
+
+```Shell
+FAILURE: Build failed with an exception.
+
+* What went wrong:
+A problem occurred configuring root project 'some_name_of_app'.
```
Comment (ikoevska) [inline troubleshooting.md]: 'some_name_of_app' -> 'app_name'

File: troubleshooting.md [hunk 3]
```diff
@@ -16,6 +16,20 @@ NativeScript {{site.current-version}} is known to have the following issues and
...
+> Could not resolve all dependencies for configuration ':_debugCompile'.
+```
+
+**Workaround** <br/>You should open android sdk manager and download Android Support Repository located in Extras section at the bottom. See [Set up windows](setup/ns-cli-setup/ns-setup-win.md)
```
Comment (ikoevska) [inline troubleshooting.md]:
```
**Workaround:** Run `$ android` to launch the Android SDK Manager and download the Android Support Repository 21.0.0.
```

**Pull request merged**